airflow/api_fastapi/app.py

```diff
@@ -43,6 +44,18 @@ def create_app() -> FastAPI:
 
     init_views(app)
 
+    origins = [
+        "http://localhost:28080",
+        "http://localhost:8080",
+    ]
+    app.add_middleware(
+        CORSMiddleware,
+        allow_origins=origins,
+        allow_credentials=True,
+        allow_methods=["*"],
+        allow_headers=["*"],
+    )
+
```

Comment:
Nice, we might want to read that from the config `api` section:
https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#access-control-allow-headers

Reply:
Also I don't know if you want to do that now or merge this one and do that in the followin PR but 'packaging' things to plug on the config as mentionned above and avoid hardcoding 'ENV' environment variable and urls.

Reply:
I am on vacation for the next week so feel free to contribute to this PR. Or merge this one and fix the urls in a follow-up PR

Reply:
Great. I just added a small commit to read the cross_origin parameters from the airflow config itself `[api]` section.

I think we're good to merge then.